avm/res/health-bot/health-bot/README.md

```diff
@@ -13,6 +13,7 @@ This module deploys an Azure Health Bot.
 - [Usage examples](#Usage-examples)
 - [Parameters](#Parameters)
 - [Outputs](#Outputs)
+- [Cross-referenced modules](#Cross-referenced-modules)
```

Comment:
Hey @AlexanderSehr, just a heads up that all orphaned modules are being converted by #3675

Reply:
Hey @eriqua, good to know, thanks.
I guess I'll assign the failed pipeline then to your PR so that not somebody else makes the same mistake.
Will close this PR